wa/workloads/antutu/__init__.py

```diff
@@ -197,7 +197,6 @@ class AntutuBDP(ApkWorkload):
 
     Known working version: 10.4.3-domesticAndroidFullBdp
     '''
-    package_name = 'com.antutu.ABenchMark'
```

Comment:
Ah good point, in which case should this not be `package_names = ['com.antutu.ABenchMark']` rather than setting this manually in `setup`?